applications/admin.py

```diff
@@ -22,3 +22,18 @@ class Media:
 @admin.register(Application)
 class ApplicationAdmin(BaseApplicationAdmin):
     inlines = [ApplicationGroupChoiceInline]
+    list_display = [
+        "name",
+        "email",
+        "get_application_groups",
+    ]
+
+    def get_application_groups(self, obj):
```

Comment:
Not important, but this is displayed as "Get application groups" in the admin panel, should rather be just "Application groups" or "Groups"